src/style.css

```
@@ -1 +1,12 @@
-/* styles go here */
+.form-field {
+  display: block;
+  margin-bottom: 10px;
+}
```

```suggestion
.form-field:not(:last-child) {
  display: block;
  margin-bottom: 10px;
}
```
That may fix your tests